Documentation/interacting_v3.md

```diff
@@ -0,0 +1,183 @@
+# Interacting with etcd
+
+Users mostly interact with etcd by putting or getting the value of a key. This section describes how to do that by using etcdctl, a command line tool for interacting with etcd server. The concepts described here should apply to the gRPC APIs or client library APIs.
+
+By default, etcdctl talks to the etcd server with the v2 API for backward compatibility. For etcdctl to speak to etcd using the v3 API, the API version must be set to version 3 via the `ETCDCTL_API` environment variable.
+
+``` bash
+export ETCDCTL_API=3
+```
+
+## Write a key
+
+Applications store keys into the etcd cluster by writing to keys. Every stored key is replicated to all etcd cluster members through raft to achieve consistency and reliability.
+
+Here is the command to set the value of key `foo` to `bar`:
+
+``` bash
+$ etcdctl put foo bar
+OK
+```
+
+## Read keys
+
+Applications can read values of keys from etcd cluster. Both ranged queries and single-key queries are supported. 
+
+Suppose we have keys already stored in etcd as blow:
+
+```
+foo = bar
+foo1 = bar1
+foo3 = bar3
+```
+
+Here is the command to read the value of key `foo`:
+
+```bash
+$ etcdctl get foo
+foo
+bar
+```
+
+Here is the command to range over the keys from `foo` to `foo9`:
+
+```bash
+$ etcdctl get foo foo9
+foo
+bar
+foo1
+bar1
+foo3
+bar3
+```
+
+## Read past version of keys
+
+Applications may want to read superseded versions of a key for various reasons. For example, an application may wish to rollback to an old configuration by accessing an early version of a key. Alternatively, an application may want a consistent view over multiple keys over multiple requests by accessing key history.
```

Comment:
s/for various reasons//
s/rollback/roll back/
s/early/earlier/
s/over multiple requests/through multiple requests/